src/modules/dashboard/components/CoinMachine/RemainingDisplayWidget/RemainingDisplayWidget.tsx

```diff
@@ -217,9 +219,13 @@ const RemainingDisplayWidget = ({
           [styles.valueWarning]: isValueWarning || showValueWarning,
         })}
       >
-        {displayedValue}
+        {saleStarted ? (
```

Comment:
we don't need this check here as this logic is already covered in `displayedValue` - line 143.

Reply:
The logic there doesn't cover the case of the remaining time widget, I'll update it tho as it should